.github/workflows/cargo-test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly-2023-05-22
+          toolchain: 1.84.1
```

Comment:
Stable version has not adopted all the linting rules we use in fmt. Cargo fmt still uses nightly and that specific version.
We can review in a later version of rust, in an sdk upgrade.
For now lets maintain the nightly.